<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QBrakeCooling</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="2">
     <widget class="QComboBox" name="brakeCategoryComboBox"/>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="taxiDistanceLabel">
      <property name="text">
       <string>Taxi Distance (miles)</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="ab2Label">
      <property name="text">
       <string>AUTOBRAKE 2</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="altitudeLabel">
      <property name="text">
       <string>Pressure Altitude</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLCDNumber" name="minutes_ab1_idle">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="18" column="2">
     <widget class="QLCDNumber" name="minutes_ab2_revt">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="22" column="0">
     <widget class="QLabel" name="cautionLabel">
      <property name="text">
       <string>Caution Zone:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="weightLabel">
      <property name="text">
       <string>Weight</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QSpinBox" name="altitudeSpinBox">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="weightSpinBox">
      <property name="minimum">
       <number>40000</number>
      </property>
      <property name="maximum">
       <number>80000</number>
      </property>
      <property name="singleStep">
       <number>500</number>
      </property>
      <property name="value">
       <number>60000</number>
      </property>
     </widget>
    </item>
    <item row="17" column="2">
     <widget class="QLCDNumber" name="minutes_ab3_revt">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="22" column="1" colspan="2">
     <widget class="QFrame" name="cautionFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QPushButton" name="calcPushButton">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item row="15" column="2">
     <widget class="QLCDNumber" name="minutes_mm_revt">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="23" column="1" colspan="2">
     <widget class="QFrame" name="warningFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="abMaxLabel">
      <property name="text">
       <string>AUTBRAKE MAX</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="ab1Label">
      <property name="text">
       <string>AUTOBRAKE 1</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLCDNumber" name="minutes_abm_idle">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="maxManualLabel">
      <property name="text">
       <string>MAX MANUAL</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="tempLabel">
      <property name="text">
       <string>OAT</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="ab3Label">
      <property name="text">
       <string>AUTOBRAKE 3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="tempSpinBox">
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLCDNumber" name="minutes_ab3_idle">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="16" column="2">
     <widget class="QLCDNumber" name="minutes_abm_revt">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="3">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="speedSpinBox">
      <property name="minimum">
       <number>80</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>140</number>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QLCDNumber" name="minutes_ab2_idle">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="20" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLCDNumber" name="minutes_mm_idle">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="23" column="0">
     <widget class="QLabel" name="warningLabel">
      <property name="text">
       <string>Fuse Plug Melt Zone:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="speedLabel">
      <property name="text">
       <string>KIAS Brake on Speed</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QSpinBox" name="TaxiDistanceSpinBox"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="coolingTimeLabel">
      <property name="text">
       <string>Cooling Time</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="idleReverseLabel">
      <property name="text">
       <string>Idle Reverse</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QLabel" name="secondDetentLabel">
      <property name="text">
       <string>2nd Detent</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="21" column="0">
     <widget class="QLabel" name="validLabel">
      <property name="text">
       <string>No Special Procedure</string>
      </property>
     </widget>
    </item>
    <item row="19" column="2">
     <widget class="QLCDNumber" name="minutes_ab1_revt">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="21" column="1" colspan="2">
     <widget class="QFrame" name="validFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="brakeCategoryLabel">
      <property name="text">
       <string>Brake Category</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>weightSpinBox</tabstop>
  <tabstop>tempSpinBox</tabstop>
  <tabstop>speedSpinBox</tabstop>
  <tabstop>altitudeSpinBox</tabstop>
  <tabstop>TaxiDistanceSpinBox</tabstop>
  <tabstop>brakeCategoryComboBox</tabstop>
  <tabstop>calcPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
